### ISSUE TITLE
Is there 10-folds training?

### ISSUE THREAD

**Issue by MjdMahasneh** [ymli39/DeepSEED-3D-ConvNets-for-Pulmonary-Nodule-Detection#25]
@ymli39 

The paper mentions 10-folds validation (which I presume, also means there is 10-fold training), my question is : 

1-how is the 10-fold training done using the training script?
2-are these files (luna_train.npy and luna_test.npy) related to the 10-fold training?
3-how are these files generated?

Looking forward to hearing back from you.

**Comment by ymli39** [comment]
Sorry about getting back to you late. I did it for 10 fold training in my paper, however for this github version, I only uploaded 1 fold as example. If you want to do a 10 fold training, you would want to generate luna_train_fold1.npy and luna_test_fold1.npy yourself and replace those with current train and test npy file. So I randomly shuffled all the list of file names and divided them in 10 group.
The order of the training and testing list does not matter that much because we focus on overall performance after 10 fold CV.
